build/DevDivVsix/EditorsPackage/Microsoft.VisualStudio.Editors.swr

```diff
@@ -0,0 +1,16 @@
+use vs
+
+package name=Microsoft.VisualStudio.Editors
+        version=15.0.0.0
```

Comment:
Having a variable is not working for some reason. Will have to revisit some other time.